geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommand.java

```diff
@@ -126,7 +128,9 @@ public String writeStacksToFile(Map<String, byte[]> dumps, String fileName) thro
       ps = new PrintWriter(os);
 
       for (Map.Entry<String, byte[]> entry : dumps.entrySet()) {
-        ps.append(STACK_TRACE_FOR_MEMBER).append(entry.getKey()).append(" ***")
+        ps.append(STACK_TRACE_FOR_MEMBER).append(entry.getKey()).append(" at ")
+            .append(new SimpleDateFormat("yyyy/MM/dd HH:mm:ss.SS").format(new Date()))
```

Comment:
This is actually using the timestamp from the JVM on which the `gfsh` command was executed after gathering the results from all members, **it is not the actual timestamp at which the thread dump was taken on each member**. Shouldn't the timestamp be obtained directly on `GetStackTracesFunction` and be part of `StackTracesPerMember`?.